marklogic-client-api/src/main/java/com/marklogic/client/impl/OkHttpServices.java

```diff
@@ -3098,7 +3104,14 @@ public Response apply(Request.Builder funcBuilder) {
       RawQueryDefinition rawQuery = (RawQueryDefinition) qdef;
       logger.debug("{} processing raw query", path);
       input = checkFormat(rawQuery.getHandle());
-    } else {
+    } else if (qdef instanceof CtsQueryDefinition) {
+        CtsQueryDefinition builtCtsQuery = (CtsQueryDefinition) qdef;
+        structure = builtCtsQuery.serialize();
+        logger.debug("{} processing cts query {}", path, structure);
+        if (sendQueryAsPayload && structure != null) {
+            input = new StringHandle(structure).withFormat(Format.TEXT);
```

Comment:
The AST for the cts.query definition is JSON, so I believe Format.JSON would be better than Format.TEXT

Reply:
Ah, yes. Changed it to JSON.